zebra-chain/src/parameters/network.rs

```diff
@@ -85,11 +170,26 @@ impl fmt::Display for Network {
 }
 
 impl Network {
+    /// Creates a new `Testnet` with no parameters.
+    pub fn new_testnet() -> Self {
+        Self::new_testnet_with_parameters(None)
+    }
+
+    /// Creates a new `Testnet` with the provided parameters.
+    pub fn new_testnet_with_parameters(parameters: impl Into<TestnetParameters>) -> Self {
```

Comment:
Except I think this one needs to be private so it can be called from both new_public_testnet() and new_configured_testnet()l

```suggestion
    fn new_testnet_with_parameters(parameters: impl Into<TestnetParameters>) -> Self {
```